clinica/iotools/utils/data_handling.py

```diff
@@ -639,3 +655,298 @@ def center_all_nifti(bids_dir, output_dir):
         for error in all_errors:
             final_error_msg += '\n' + error
         raise RuntimeError(final_error_msg)
+    return nifti_files_filtered
+
+
+def write_list_of_files(file_list, output_file):
```

Comment:
Needs at least a small description e.g. :
```suggestion
def write_list_of_files(file_list, output_file):
"""Save `file_list` list of files into `output_file` text file."""
```

Reply:
Done